dockertest/environment.py

```diff
@@ -62,3 +65,13 @@ def get_selinux_context(path):
     """
     # First list item is null-terminated string length
     return selinux.getfilecon(path)[1]
+
+
+def selinux_is_enforcing():
+    """
+    Return True if selinux is currently in enforcing mode.
+    """
+    # Travis complains about missing selinux.ENFORCING
+    # pylint: disable=C0103
+    ENFORCING = 1  # Travis complains about missing selinux.ENFORCING
+    return selinux.security_getenforce() == ENFORCING
```

Comment:
I'm still not sure what the purpose is of this selinux module, but yes, that's better. Thanks especially for the link to actual documentation.

And speaking of documentation, since security_getenforce(3) just says "returns 0 / 1 / -1" (not named constants), why not remove the entire ENFORCING constant, and comment, and pylint, and just make it `== 1`?

And to make it even cleaner, rename this function to `selinux_getenforce` and change the `failif` below to `failif_ne(selinux_getenforce, 1, ...)` and that way you get to differentiate between 0, 1, and the not-expected but you-never-know -1.